netflix-roulette/src/store/interfaces/commonInteraces.ts

```diff
@@ -26,5 +26,5 @@ export interface MoviesState extends AppState {
 
 export interface MoviesActions {
   type: string;
-  payload: any;
+  payload?: any;
```

Comment:
`any` is never good idea, I guess, `unknown` would be even better option, other than that, you may have `MoviesActions` derive from specific actions, e.g.: 
```ts
const SomeMovieAction = ({somePayload}) => ({
   type: MovieActionType,
   somePayload
})

const SomeOtherMovieAction = ({someOtherPayload}) => ({
  type: MovieActionType,
  someOtherPayload
})

..

interface MovieActions = ReturnType<typeof SomeMovieAction>
  |  ReturnType<typeof SomeOtherMovieAction>
  | ..
```